### PR TITLE
add industries grid component

### PR DESCRIPTION
**Pull request by jesicasusanto**
Resolves Issue #7 
This PR 
- implements the industries grid component of the web page that is responsive to the screen size
- fixed the zoomed in screen when first opening the web page

- Before :

image

image

- After :

image

image

steps to reproduce 
same as [Getting Started](https://github.com/OpenAdaptAI/web#getting-started) in readme

### CODE REVIEW

**Comment by jesicasusanto** [comment]
Made the cards responsive to screen size
Before :
image

After :
image

**Comment by jesicasusanto** [comment]
Change cards color to be dark purple then add card shadows.
Before :
image
After :
image

Would love to get feedback on this @abrichr . Thank you!

**Comment by dianzrong** [comment]
@jesicasusanto I really like the shadows! Do you think the card colours should be lighter to improve visibility?

**Comment by jesicasusanto** [comment]
@dianzrong thank you! i agree too. I've made the grids background lighter.

**Comment by jesicasusanto** [comment]
Ready for review @abrichr @Jonathan-Sendsen.

Summary :
- added industry grids
- added value proposition

**Comment by abrichr** [comment]
Please add "(For Developers)" after "Getting Started"

**Comment by abrichr** [comment]
Please modify grid styles so that padding is symmetrical horizontally/vertically between items

**Comment by abrichr** [comment]
@jesicasusanto can you please add an icon to each one, e.g. from https://thenounproject.com/ ? 🙏

**Comment by abrichr** [comment]
https://openadapt.ai/ :

image

https://deploy-preview-15--cosmic-klepon-3c693c.netlify.app/:

image


Can you please restore the padding / margins around "AI for Humans", "Automate your workflows", "No programming required"?

**Comment by abrichr** [comment]
Also please feel free to implement https://github.com/OpenAdaptAI/web/issues/17 while you're at it 😄

**Comment by jesicasusanto** [comment]
Hi @abrichr , thank you for the feedback.
I have made the changes according to your feedback :
- restored the padding/margins
- restored the Get Started button and link it to industries grid
- made the buttons secondary
For the logos, I feel like we need them to be white logos which is only included in the paid package from https://thenounproject.com/. Will this be okay with you? Please let me know what you think! Thank you.

**Comment by abrichr** [comment]
For each of the industries, please mention specific use cases / roles / applications. You can find these in the issues.

e.g.:
> Optimize patient care and medical record keeping by integrating our automation tool, freeing healthcare professionals for critical tasks.

This should mention specifics, e.g.:

> Automate Revenue Cycle Management, Auditing, and Encounter Documentation in Cerner, Epic, and Excel, and more.

Please assemble a list of these descriptions in a Google Doc that we can collaborate on. Include links to specific Github comments and other sources 🙏

File: components/IndustriesGrid.js [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+      buttonLabel: 'Join Waitlist',
+      logo: '/images/noun-build.svg',
+    },
+    {
+      title: 'Developers',
+      descriptions: 'Simplify software development processes',
+      buttonLabel: 'Join Waitlist',
+      logo: '/images/noun-developer.svg',
+    },
+  ];
```
Comment (abrichr) [inline components/IndustriesGrid.js:67]: Excellent thank you for defining these as objects! 🙏

**Comment by jesicasusanto** [comment]
Hi @abrichr for cropping the logos to content, gimp doesn't support exporting the logo back to an svg file. As an alternative, I adjusted the width and height attributes inside the svg files according to the heightxwidth sized given by the gimp "crop to content" command. Please let me know what you think of this solution. Thank you! 
Before :

image

After :
image

**Comment by abrichr** [comment]
Thanks @jesicasusanto !

Can you please remove the developer grid item? Then modify the "Getting Started" header to "For Developers".

**Comment by jesicasusanto** [comment]
Update @abrichr 
- [x] remove the developer grid item (addressed in https://github.com/OpenAdaptAI/web/pull/15/commits/00b66e13d933d4e89e9b2fe5fb71994861bbc5b4)
- [x] modify the "Getting Started" header to "For Developers" (addressed in https://github.com/OpenAdaptAI/web/pull/15/commits/c54e26ced9749e56aa6ff32f5bbcab8e4e9d1604)
- [x] remove this heading altogether:
Revolutionizing Industries with AI-First Automation for Growth and Transformation. (addressed in https://github.com/OpenAdaptAI/web/pull/15/commits/0f44787c392cb6d6c36b6d749f6b174d12dab0fa)
- [x] rename "Freight" to "Logistics" (addressed in https://github.com/OpenAdaptAI/web/pull/15/commits/3eae0bfe3f224e4f821b8851ea27f5c238b5520f).
- [x] center grid content vertically (addressed in https://github.com/OpenAdaptAI/web/pull/15/commits/d52ac596e2cf02818c92559954f1e8fa761d2521)

I also added a silver border around the cards when we hover them as you suggested on our last sprint in https://github.com/OpenAdaptAI/web/pull/15/files#diff-4130362d983b52ff81285caab923910466ada416e0ca9974ad2ffd36f674366e:~:text=%7D-,.card%3Ahover%20%7B,-border%3A%202 . But Im not sure if this aligns with your vision for the hover card animation. Would love to get a clarification in this. Looking forward to your feedback!

**Comment by abrichr** [comment]
Hi @jesicasusanto :

- It looks like the hover animation on the icons is no longer working. Can you please fix it?
- The border is too bright. Please make it much more subtle. It should be barely perceptible. It should also be animated.
- The background color of the grid component does not appear to change on hover. It should transition to a slightly brighter color over e.g. 0.5s.

Please let me know if anything is unclear!

**Comment by abrichr** [comment]
@jesicasusanto the updated border color looks better, and the background color animation looks good 👍 

Todo:
- fix icon animation (make sure you are opening the page in incognito mode and/or clearing your cache to reproduce the issue. Right now icons just switch between small and large with no animation)
- set the "Join Waitlist" border color to transparent on hover rather than removing it altogether. This is necessary to prevent the rest of the grid content from moving down when the user hovers with their mouse due to the reduced size from the border being removed.

Please let me know if anything is unclear!

**Comment by jesicasusanto** [comment]
Update @abrichr :

- [x] added margin around logo (addressed in https://github.com/OpenAdaptAI/web/pull/15/commits/9b212d65ad1bd93b222b31d37d2d3a704bc84245)
- [x] added outline when the industry grid is hovered (addressed in https://github.com/OpenAdaptAI/web/pull/15/commits/302fab44c67eb70db1938730c26edcbf8f94dd85)

**Pull request closed without merging**